### PR TITLE
Remove detached

### PR DESCRIPTION
**Pull request by mhevery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/core/src/render3/instructions.ts [hunk 1]
```diff
@@ -294,28 +297,28 @@ export function allocExpando(view: LView) {
  */
 export function renderTemplate<T>(
     hostNode: RElement, templateFn: ComponentTemplate<T>, consts: number, vars: number, context: T,
-    providedRendererFactory: RendererFactory3, hostView: LView | null,
+    providedRendererFactory: RendererFactory3, componentView: LView | null,
```
Comment (benlesh) [inline packages/core/src/render3/instructions.ts]: nit: should be `componentLView`

File: packages/core/src/render3/instructions.ts [hunk 2]
```diff
@@ -1949,13 +1952,13 @@ function addComponentLogic<T>(
       lView, previousOrParentTNode.index as number,
       createLView(
           lView, tView, null, def.onPush ? LViewFlags.Dirty : LViewFlags.CheckAlways,
+          lView[previousOrParentTNode.index], previousOrParentTNode as TElementNode,
           rendererFactory, lView[RENDERER_FACTORY].createRenderer(native as RElement, def)));
 
-  componentView[HOST_NODE] = previousOrParentTNode as TElementNode;
+  componentView[T_HOST] = previousOrParentTNode as TElementNode;
```
Comment (benlesh) [inline packages/core/src/render3/instructions.ts]: same as above, `componentLView` (or `componentTView`, I'm not actually sure here without an IDE to check)

File: packages/core/src/render3/node_manipulation.ts [hunk 3]
```diff
@@ -441,7 +442,7 @@ function cleanUpView(viewOrContainer: LView | LContainer): void {
 
     executeOnDestroys(view);
     removeListeners(view);
-    const hostTNode = view[HOST_NODE];
+    const hostTNode = view[T_HOST];
```
Comment (benlesh) [inline packages/core/src/render3/node_manipulation.ts]: `view` variable name needs to be more specific. The argument doc comment for this function is also out of date.

File: packages/core/src/render3/node_manipulation.ts [hunk 4]
```diff
@@ -525,7 +526,7 @@ function getRenderParent(tNode: TNode, currentView: LView): RElement|null {
   // If the parent is null, then we are inserting across views: either into an embedded view or a
   // component view.
   if (parent == null) {
-    const hostTNode = currentView[HOST_NODE] !;
+    const hostTNode = currentView[T_HOST] !;
```
Comment (benlesh) [inline packages/core/src/render3/node_manipulation.ts]: currentLView

File: packages/core/src/render3/node_manipulation.ts [hunk 5]
```diff
@@ -568,7 +569,7 @@ function getRenderParent(tNode: TNode, currentView: LView): RElement|null {
  * a host element.
  */
 function getHostNative(currentView: LView): RElement|null {
```
Comment (benlesh) [inline packages/core/src/render3/node_manipulation.ts]: currentLView

File: packages/core/src/render3/node_manipulation.ts [hunk 6]
```diff
@@ -691,7 +692,7 @@ function getHighestElementOrICUContainer(tNode: TNode): TNode {
 export function getBeforeNodeForView(index: number, views: LView[], containerNative: RComment) {
   if (index + 1 < views.length) {
     const view = views[index + 1] as LView;
-    const viewTNode = view[HOST_NODE] as TViewNode;
+    const viewTNode = view[T_HOST] as TViewNode;
```
Comment (benlesh) [inline packages/core/src/render3/node_manipulation.ts]: lView and lViews

**Review by benlesh — CHANGES_REQUESTED** [review]
Mostly just nits about naming. I think we should be putting forth the effort to name views more specifically, because while I'm getting more familiar with the code, I'm still not sure if I'm looking at a TView or LView, for example, when I'm reading little snippets on Github PRs and I'm not in my IDE.

We can change them later, I suppose, but I feel like I keep saying that.

**Comment by ocombe** [comment]
since https://github.com/angular/angular/pull/28520 I think that we can close this PR?

**Review by benlesh — APPROVED** [review]

**Comment by mhevery** [comment]
@ocombe The other one did not land, I closed it. This is the one we should land as I don't think the behavior is needed and it passes material.

**Pull request closed without merging**